andrew/euler/euler_2.py

```diff
@@ -0,0 +1,16 @@
+'''
+Consider the terms in the Fibonacci sequence whose values do not exceed four million.
+Find the sum of the even valued terms
+'''
+
+
+prev, after = 0, 1
+total = 0
+while True:
```

Comment:
Try doing it with a generator expression. Something that yields infinitely many fibonacci number, then use filter and takewhile (in itertools).